docs/includes/excel-custom-functions-note.md

```diff
@@ -1,2 +1,2 @@
 > [!IMPORTANT]
-> Note that Excel custom functions are available in Office on Windows (version 1904 or later, connected to Office 365 subscription), Office on Mac (version 16.24 or later, connected to Office 365 subscription), and Office on the web. Excel custom functions are not supported in Office 2019 (one-time purchase).
+> The custom functions feature is being released to production! Discover the latest updates to custom functions [in this blog post](https://developer.microsoft.com/en-us/office/blogs/office-extensibility-build-2019/). Note that Excel custom functions are available in Office on Windows (version 1904 or later, connected to Office 365 subscription), Office on Mac (version 16.24 or later, connected to Office 365 subscription), and Office on the web. Excel custom functions are not supported in Office 2019 (one-time purchase).
```

Comment:
Aren't CFs in production? If so, change the first sentence to match that reality (is in production).

Reply:
So apparently not